ui/round/css/_app-layout.scss

```diff
@@ -60,17 +60,13 @@
     }
 
     .rclock-bottom {
-      grid-area: 7 / 1 / 9 / 2;
+      grid-area: 10 / 1 / 8 / 2;
```

Comment:
what is this about?

Reply:
as I have added 1 row for `'expi-top'` area the clock should be placed between rows 8 to 10 instead of 7 to 9 therefore I added this change.